docs/topics/vscode-extension-configuring-analysis.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * docs/vs-code-extension-guide/master.adoc
+
+[id='vscode-extension-configuring-analysis_{context}']
+= Configuring a run configuration
+
+You can configure multiple run configurations to run against each project.
+
+.Prerequisites
+
+* Projects imported into VS Code.
+
+.Procedure
+
+. In *Extensions* view, click the *{ProductName}* icon (image:vs_mta_ex_icon.png[MTA code extension]) on the Activity bar.
+. Click the *+* (plus sign) next to *{ProductName}* to add a run configuration.
+. Complete the following configuration fields:
+
+* *Input*: Click *Add* and do one of the following:
+
+** Enter the input file or directory and press Enter.
+** Click *Open File Explorer* and select the directory.
+
+* *Target*: Select one or more target migration paths.
+
+. Optional: Configure any advanced options needed, such as excluding specific packages from analysis or disabling Tattletale reports.
```

Comment:
How would the user know how to do this? I think there is a tab or a section in the UI. You should at least mention how advanced options are configured. Also, an example would help.

. To exclude packages from analysis, click (or whatever user is supposed to do) the Advanced options tab and select "X".

Reply:
It's all one pane, no additional tabs. 
![image](https://user-images.githubusercontent.com/60698649/133063071-f140ff79-69ea-4b42-aef8-e4f61dd498f6.png)

Reply:
In that case, it is pretty obvious. I do not think you need to document this, not even as an optional step.